src/Nest/Mapping/Types/Geo/GeoShape/GeoOrientation.cs

```diff
@@ -1,16 +1,45 @@
-﻿using System.Runtime.Serialization;
+﻿using System;
 using Newtonsoft.Json;
-using Newtonsoft.Json.Converters;
 
 namespace Nest
 {
-	[JsonConverter(typeof(StringEnumConverter))]
+	[JsonConverter(typeof(GeoOrientationConverter))]
 	public enum GeoOrientation
 	{
-		[EnumMember(Value = "cw")]
 		ClockWise,
-
-		[EnumMember(Value = "ccw")]
 		CounterClockWise
 	}
+
+	internal class GeoOrientationConverter : JsonConverter
+	{
+		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+		{
+			var geoOrientation = (GeoOrientation)value;
+			switch (geoOrientation)
+			{
+				case GeoOrientation.ClockWise:
+					writer.WriteValue("cw");
+					break;
+				case GeoOrientation.CounterClockWise:
+					writer.WriteValue("ccw");
+					break;
+			}
+		}
+
+		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+		{
+			var enumString = (string)reader.Value;
+			switch (enumString.ToLower())
```

Comment:
think this should use `ToLowerInvariant()`, rather than use the current culture

Reply:
Didn't make the cut for the merge, but will fix for `7.x`